platform/stellar/transaction.go

```diff
@@ -19,42 +17,17 @@ func (p *Platform) GetTxsByAddress(address string) (blockatlas.TxPage, error) {
 }
 
 func (p *Platform) NormalizePayments(payments []Payment) (txs []blockatlas.Tx) {
```

Comment:
func (p *Platform) NormalizePayments(payments []Payment) (txs []blockatlas.Tx) {

TO

func (p *Platform) NormalizePayments(payments []Payment) []blockatlas.Tx {
txs := make([]blockatlas.Tx, 0, len(payments))

?

Reply:
Done